app/views/content/landing/grow/_content.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Career progression</h2>
+
+    <div class="inset">
+      <p>
+      Whether you want to climb the ladder into leadership roles, or focus on the areas that motivate you, life as a teacher offers varied and exciting prospects. There are lots of ways for you to develop your career, each bringing its own rewards and challenges. 
+      </p>
+
+      <p>
+      You can choose your own path and:
+      </p>
+
+      <ul>
+      <li>use your passion about the subject you teach to help shape the curriculum</li> 
+      <li>support pupil wellbeing, for example work with children with special educational needs (SEN), or become a form tutor</li> 
+      <li>become a headteacher and inspire the teachers and pupils at your school</li> 
+      </ul>
+
+      </p>
+
+      <p><%= link_to("Find out more about career progression", page_path("is-teaching-right-for-me/teacher-pay-and-benefits")) %>.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Personal development</h2>
+
+    <div class="inset">
+      <p>
+      As a teacher your learning never stops. Teaching is a career that encourages you to grow, both personally and professionally. Not only can you make a difference to the lives of the pupils you teach, you can make a difference to your own. 
+      </p>
+
+      <p>
+      If your teacher training course leads to qualified teacher status (QTS), you get a whole package of training and support as an early career teacher (ECT). This includes: 
+      </p>
+
+      <ul>
+      <li>regular one-to-ones with a mentor to help you boost your confidence</li>
+      <li>learning new skills</li>
+      <li>guidance through any challenges you might have</li>
+      </ul>
+
+      </p>
+
+      <p><%= link_to("Find out more about support and development for early career teachers (ECTs)", page_path("support-for-early-career-teachers")) %>.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Opportunities to increase your pay</h2>
+
+    <div class="inset">
+      <p>
+      With a competitive starting salary of £30k (or more in London), it pays to do what you love. And there are plenty of opportunities for you can increase your pay as you progress your career:  
```

Comment:
@MaxineCl Second sentence needs rewording - doesn't quite make sense!

Reply:
I've removed can as I think it was in there by mistake!